webapp/src/Utils/Scoreboard/ProblemSummary.php

```diff
@@ -7,82 +7,47 @@ class ProblemSummary
     /**
      * @var int[]
      */
-    protected $numberOfSubmissions = [];
+    public $numSubmissions = [];
 
     /**
      * @var int[]
      */
-    protected $numberOfPendingSubmissions = [];
+    public $numSubmissionsPending = [];
```

Comment:
numPendingSubmissions?

Reply:
I preferred this way since I think it's clear and grammatically correct too, and now things align better:
```php
$numSubmissions
$numSubmissionsPending
$numSubmissionsCorrect
```